### PR TITLE
Fix: NoSuchFieldException in TeslaSWTAccess

### PR DESCRIPTION
**Pull request by pzi-dsa**
The messageCount field of class org.eclipse.swt.widgets.Synchronizer no
longer exists. Use messages.size() instead.

See [here](https://github.com/eclipse-platform/eclipse.platform.swt/commit/b4030cbc1fe03d8dd01635dbfbb1b9b7bf6d707b#diff-005117e4747b047ea1c7cd63afb46e29b59197508152c0b953a1a0bb4c07fa59) for the corresponding commit.

Fixes [Bug 580108](https://bugs.eclipse.org/bugs/show_bug.cgi?id=580108) and [Bug 580265](https://bugs.eclipse.org/bugs/show_bug.cgi?id=580265).

### CODE REVIEW

File: runtime/tesla/org.eclipse.rcptt.tesla.swt/src/org/eclipse/rcptt/tesla/internal/ui/player/TeslaSWTAccess.java [hunk 1]
```diff
@@ -1,19 +1,21 @@
 /*******************************************************************************
- * Copyright (c) 2009, 2019 Xored Software Inc and others.
+ * Copyright (c) 2009, 2019, 2022 Xored Software Inc and others.
```
Comment (eselmeister) [inline runtime/tesla/org.eclipse.rcptt.tesla.swt/src/org/eclipse/rcptt/tesla/internal/ui/player/TeslaSWTAccess.java:2]: Better change to:
`Copyright (c) 2009, 2022 Xored Software Inc and others.`

Comment (pzi-dsa) [reply runtime/tesla/org.eclipse.rcptt.tesla.swt/src/org/eclipse/rcptt/tesla/internal/ui/player/TeslaSWTAccess.java]: Done.

File: runtime/tesla/org.eclipse.rcptt.tesla.swt/src/org/eclipse/rcptt/tesla/internal/ui/player/TeslaSWTAccess.java [hunk 2]
```diff
@@ -526,9 +528,9 @@ public static Menu getCTabFolderItemListMenu(CTabFolder tabFolder) {
 
 	public static int getRunnables(Synchronizer synchronizer) {
 		try {
-			Field locationField = Synchronizer.class.getDeclaredField("messageCount");
```
Comment (haiodo) [inline runtime/tesla/org.eclipse.rcptt.tesla.swt/src/org/eclipse/rcptt/tesla/internal/ui/player/TeslaSWTAccess.java:529]: Do we look compatibility with older versions of Eclipse?

Comment (vogella) [reply runtime/tesla/org.eclipse.rcptt.tesla.swt/src/org/eclipse/rcptt/tesla/internal/ui/player/TeslaSWTAccess.java:529]: FrameworkUtil.getBundle(CTabFolder.class).getBundleId() should allow you @pzi-dsa to check the version of the class and only change the field if the version is after the changed field name.

**Comment by pzi-dsa** [comment]
The formatting was a little off, so I had both tabs and spaces in the commit. I've cleaned that one up, as well.

**Review by haiodo — APPROVED** [review]

**Pull request merged**